### PR TITLE
Delete the need for special node version

### PR DESCRIPTION
**Pull request by witchent**

### CODE REVIEW

**Comment by nr23730** [comment]
First of all: Thank you very much for updating my signal-desktop mod to the latest version! After the initial work was done I lost the focus a bit. 

Regarding the required node version: I experienced some trouble in the past with "other" versions. I'm okay with removing the requirement for that specific version but I would like to keep a hint that explains why 12.13.0 is the best bet to get everything working.

**Comment by witchent** [comment]
Sure, do you want to edit this? Or should I add the notice? Feel free to make any adjustments you deem worthy.

Also, there are some more adjustments I need to make to make it work again (the internal changed quite a lot, uuids got added, read messages are handled a bit different, functions changed their names and expect different input etc). I am in the process of getting everything running again. And as I don't want to have more breaking changes I want to first get everything working before pushing them to you.

Also, I am working upstream with the developer of node-webcrypto-ossl to see if we can get the upstream version to work with signal-client. This might be a bit further away though.

So to sum it up: Maybe wait a few days so I can patch everything up.

**Comment by nr23730** [comment]
Yeah, I can do so. 

Take your time to make the adjustments and provide pull requests then. If you have any questions feel free to ask. 

Also pretty cool that you're in contact with node-webcrypto-ossl. Back then I just made the fork because it was kind of outdated either way and I was hoping that future node versions will have the feature integrated.
